### ISSUE TITLE
Bigtable:  new system test failure

### ISSUE THREAD

**Issue by tseaver** [GoogleCloudPlatform/google-cloud-python#5593]
#5513 caused a new (non-flaky, AFAICT) [system test failure](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-python/7142):

```python

    def test_create_table_with_split_keys(self):
        temp_table_id = 'foo-bar-baz-split-table'
        initial_split_keys = [b'split_key_1', b'split_key_10',
                              b'split_key_20', b'']
        temp_table = Config.INSTANCE.table(temp_table_id)
        temp_table.create(initial_split_keys=initial_split_keys)
        self.tables_to_delete.append(temp_table)
    
        # Read Sample Row Keys for created splits
        sample_row_keys = temp_table.sample_row_keys()
        self.assertEqual(set([srk.row_key for srk in sample_row_keys]),
>                        set(initial_split_keys))
E       AssertionError: Items in the second set but not the first:
E       'split_key_1'
E       'split_key_10'
E       'split_key_20'
```

**Comment by aneepct** [comment]
@tseaver I fixed in my brach feature/initial_split_keys_on_table

**Comment by tseaver** [comment]
@aneepct The new system test added in PR #5513 is the one which is failing (it fails repeatably for me on my local machine, too).

**Comment by aneepct** [comment]
@tseaver I am going to create new PR for this.

**Comment by sduskis** [comment]
@aneepct, thanks!

**Comment by aneepct** [comment]
@tseaver Fixed the system test at #5606
